src/main/resources/view/RentalInformationListPanel.fxml

```diff
@@ -1,10 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.control.ListView?>
-<?import javafx.scene.layout.VBox?>
-<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
```

Comment:
Same comment as in `MainWindow.fxml` line 3:

> Should these be explicit imports instead of wildcard imports?
>
> I think that using explicit imports here would improve readability and clarity.